From d871aa864e77d89f95abe22e09d1f4bf1bc9c92e Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Tue, 10 Sep 2024 14:02:38 +0800
Subject: [PATCH 2/6] configurations: nvidia_io_board: recalculate bus number

Change the bus number calculation to meet sensor naming.

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: Ie09a73bd97a5a28d91bbaaba7e7b03f46b56a697
---
This patch will remain in facebook openbmc.
---
 configurations/nvidia_gb200_io_board.json | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/configurations/nvidia_gb200_io_board.json b/configurations/nvidia_gb200_io_board.json
index 1293c78..3ba605a 100644
--- a/configurations/nvidia_gb200_io_board.json
+++ b/configurations/nvidia_gb200_io_board.json
@@ -3,17 +3,17 @@
         {
             "Address": "$address",
             "Bus": "$bus",
-            "Name": "NVIDIA GB200 IO Board $bus FRU",
+            "Name": "NVIDIA GB200 IO Board $bus - 29 % 11 FRU",
             "Type": "EEPROM"
         },
         {
             "Address": "0x1f",
-            "Bus": 36,
+            "Bus": "$bus - 5",
             "Labels": [
                 "temp2"
             ],
-            "Name": "IOB$bus_NIC0_LOCAL_TEMP_C",
-            "Name1": "IOB$bus_NIC0_TEMP_C",
+            "Name": "IOB$bus - 29 % 11 NIC0_LOCAL_TEMP_C",
+            "Name1": "IOB$bus - 29 % 11 NIC0_TEMP_C",
             "PowerState": "On",
             "Thresholds": [
                 {
@@ -42,12 +42,12 @@
         },
         {
             "Address": "0x1f",
-            "Bus": 38,
+            "Bus": "$bus - 3",
             "Labels": [
                 "temp2"
             ],
-            "Name": "IOB$bus_NIC1_LOCAL_TEMP_C",
-            "Name1": "IOB$bus_NIC1_TEMP_C",
+            "Name": "IOB$bus - 29 % 11 NIC1_LOCAL_TEMP_C",
+            "Name1": "IOB$bus - 29 % 11 NIC1_TEMP_C",
             "PowerState": "On",
             "Thresholds": [
                 {
@@ -75,7 +75,7 @@
             "Type": "TMP421"
         }
     ],
-    "Name": "NVIDIA GB200 IO Board $bus",
+    "Name": "NVIDIA GB200 IO Board $bus - 29 % 11",
     "Probe": [
         "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'P4768', 'BOARD_MANUFACTURER': 'NVIDIA'})",
         "OR",
-- 
2.31.1

